### PR TITLE
chore(deps): Update deps to get rid of security warnings.

### PR DESCRIPTION
**Pull request by shane-tomlinson**

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
cc @vladikoff

**Comment by shane-tomlinson** [comment]
fixes #677, #678

File: packages/fxa-oauth-console/app/adapters/application.js [hunk 1]
```diff
@@ -129,16 +128,24 @@ export default DS.RESTAdapter.extend({
...
    */
-  buildURL: function(type, id, record) {
+  buildURL: function(type, id, record, requestType) {
     var url = [];
     var host = this.host;
     var prefix = this.urlPrefix();
 
-    // FxA OAuth API requires singular 'client' when the record id is set
-    if (record) {
+    switch (requestType) {
```
Comment (shane-tomlinson) [inline packages/fxa-oauth-console/app/adapters/application.js]: This seems to be the `Ember way` of doing things: https://github.com/emberjs/data/blob/v3.8.0/addon/-private/adapters/build-url-mixin.js#L56

**Comment by dannycoates** [comment]
woof, 4.8.2 is old.

**Comment by shane-tomlinson** [comment]
Closing this, we can come back to it another day.

**Pull request closed without merging**